torchinfo/torchinfo.py

```diff
@@ -150,7 +150,11 @@ def summary(
     hooks: Optional[List[RemovableHandle]] = [] if input_data_specified else None
     idx: Dict[int, int] = {}
     named_module = (model.__class__.__name__, model)
-    apply_hooks(named_module, model, batch_dim, depth, summary_list, idx, hooks)
+    # Max number of nested layers to traverse when calculating Mult-Adds. Always calculate all layers
+    calculate_depth = 1e6
```

Comment:
Depth is used for a different purpose - depth tells torchinfo when to stop showing nested layers. You should add a new parameter to avoid breaking existing functionality.